lib/TableUtils.lua

```diff
@@ -18,9 +18,14 @@ function TableUtils.GetLength(T) --: (table) => number
 	return count
 end
 
-function TableUtils.Assign(target, source)
-	for key, value in pairs(source) do
-		target[key] = value
+function TableUtils.Assign(target, ...)
```

Comment:
Nice check on the source ~= nil but have a look at the version I made on hugh-util-tools - I think it's a bit more concise not using select